src/components/date-picker/date-picker.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import {Dropdown} from './dropdown';
+import {observable, action} from 'mobx';
+import {observer} from 'mobx-react';
+import {SyntheticEvent} from "react";
+import {KeyCodes} from '../../common/key-codes';
+import styles from './date-picker.st.css';
+
+const invalidDate: string = 'Invalid Date';
+
+export interface DatePickerProps {
+    value: Date;
+    onChange (value: Date): void;
+    isDropdownVisible?: boolean;
+    placeholder?: string;
+}
+
+@observer
```

Comment:
mobx here is used only for simple flag/value storing, do we really need it here?